packages/components/Tabs/src/TabsItem.settings.ts

```diff
@@ -0,0 +1,95 @@
+import { tabsItemName, TabsItemType } from './TabsItem.types';
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import type { IViewProps } from '@fluentui-react-native/adapters';
+
+export const tabsItemSelectActionLabel = 'Select a TabsItem';
+
+export const settings: IComposeSettings<TabsItemType> = [
+  {
+    tokens: {
+      color: 'neutralForeground3Brand',
+      variant: 'bodyStandard',
+      borderWidth: 2,
+      borderColor: 'transparent',
+      borderRadius: 4,
+      indicatorColor: 'transparent',
+    },
+    root: {
+      accessible: true,
+      focusable: true,
+      accessibilityRole: 'tab',
+      style: {
+        display: 'flex',
+        alignItems: 'center',
+        flexDirection: 'column',
+        alignSelf: 'flex-start',
+        justifyContent: 'center',
+      },
+    } as IViewProps,
+    indicator: {
+      style: {
+        minHeight: 2,
```

Comment:
Have you tested this at all DPIs?

Reply:
No we haven't, when you say DPI do you mean dots per inch? Also how would we go about testing it.

Reply:
Yes, dots per inch. You can easily change it in your computer settings. I would try each configuration and just visually make sure this component looks normal.

Reply:
We tried it and it looks good!